Cargo.toml

```diff
@@ -15,6 +15,8 @@ edition = "2021"
 
 [dependencies]
 num-traits = "0.2.19"
+rand_distr = "0.4"
```

Comment:
This is cool, but not a core dependency of an n dimensional array, more an example